migrations/20231202002710_new_tables.mjs

```diff
@@ -0,0 +1,89 @@
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async (knex) => {
+  
+ knex.schema.withSchema('public').createTable('victims', (table) => {
+  table.increments('victim_id', { primaryKey: true }).notNullable();
+  table.date('victim_creation');
+  table.date('victim_last_update');
+  table.specificType('victim_name_lastname', 'character varying');
+  table.integer('victim_age');
+  table.string('victim_nationality', 3); 
+  table.boolean('victim_prostitution');
+  table.boolean('victim_missing');
+  table.boolean('victim_native_people');
+  table.boolean('victim_pregnant');
+  table.boolean('victim_disabillity');
+  table.text('victim_ocupation');
+  table.boolean('victim_children');
+ });
+
+
+ knex.schema.withSchema('public').createTable('aggresors', (table) => {
+  table.increments('aggresor_id', { primaryKey: true }).notNullable();
+  table.date('aggresor_creation');
+  table.date('aggresor_last_update');
+  table.specificType('aggresor_gender', 'character varying');
+  table.specificType('aggresor_name_lastname', 'character varying');
+  table.integer('aggresor_age');
+  table.boolean('aggresor_legal_complaint_history');
+  table.boolean('aggresor_cases_history');
+  table.boolean('aggresor_captive_history');
+  table.boolean('aggresor_behaviour_post_case');
+ });
+
+
+knex.schema.withSchema('public').createTable('cases', (table)=>{
+  table.increments('case_id', { primaryKey: true}).notNullable();
+  table.foreign('victim_id').references('victims.victim_id');
+  table.foreign('aggresor_id').references('aggresors.aggresor_id');
+  table.date('case_creation');
+  table.date('last_update');
```

Comment:
Deberían ser NOT NULL. Rompiste la convención acá, estás poniendo `case_campo` en todos lados excepto en `last_update` e `incident_date`.